test-wrapper/src/main.rs

```diff
@@ -0,0 +1,180 @@
+use std::env;
```

Comment:
also some toplevel comment please, what's in here.

if there is no separate docs file, maybe also motivation / usage.